extensions/blocks/donations/attributes.js

```diff
@@ -38,7 +45,7 @@ export default {
 	},
 	chooseAmountText: {
 		type: 'string',
-		default: __( 'Choose an amount (USD)', 'jetpack' ),
+		default: __( 'Choose an amount', 'jetpack' ) + ' (USD)',
```

Comment:
I think we should avoid to do something like this, as it is going to make it difficult to translate in other languages.

Reply:
Well, this change was done to deliberately leave the currency code out of the translation, since the code is actually international and non-translatable. This also ensures the default attribute value always contains `(USD)`, regardless the translation, making possible to replace the currency whenever it changes if a user is still using the default attribute value: 

https://github.com/Automattic/jetpack/blob/56c2599a8d134b432c8df9f7113cb3028efb2622/extensions/blocks/donations/tabs.js#L86

> it is going to make it difficult to translate in other languages

Would you mind expanding that with some examples? Assuming it's ok to not translate the currency code, the other 2 things this change produces are:
- Currency is wrapped by parenthesis.
- Currency is appended to the end of the text.

Could any of those be an issue in other languages?

Reply:
We'd still drop that in as a variable within the string I think, for example if translators need to reorder. In this case I'm not sure if the currency code needs to be visible here, but I'll also cc @Automattic/i18n for recommendations.

Reply:
Yes, please introduce a variable here. In some languages (I can think of German), it'd be better style to not put the code in parens but embed it in a sentence (something like `"Choose an amount in %(currency)s"` in English). Additionally, it would be nice to not force the currency codes on users, in Europe we prefer using € instead of EUR. Maybe we can make them translatable, too.

Reply:
+1. Noting that you can now use `jetpackCreateInterpolateElement` to build the string you need:
https://github.com/Automattic/jetpack/blob/bc9d081033570eba40b9c6666379c725355e362e/extensions/README.md#i18n

Reply:
Finally opted for not including the currency in the "choose amount" label by default since it's redundant (the currency already appears below in the buttons).